specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
In the future, we could work with the `county_contest` info that we actually have to make up a `contest` table which includes all `county_contests` that a particular contest involves, matching on the contest `name`.
We've been working on them for months, as have they, to ensure that the naming of contests is uniform across counties. We'll see if that actually works....